libsigner/src/events.rs

```diff
@@ -73,6 +74,18 @@ pub enum SignerEvent {
     BlockValidationResponse(BlockValidateResponse),
     /// Status endpoint request
     StatusCheck,
+    /// A new burn block event was received with the given burnchain block height
+    NewBurnBlock(u64),
+}
+
+/// A struct to aid in deserializing the new burn block event
+#[derive(Debug, Deserialize)]
+struct TempBurnBlockEvent {
```

Comment:
If this is only used in `process_new_burn_block_event()`, you could potentially move it to within the function body so it doesn't look like this struct is needed in the module itself.

Reply:
Will do!